src/backend/Functions/FunctionsDIConfiguration.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using DependencyInjection;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Host.Config;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Fx.Portability.Azure.Storage;
+using Microsoft.Fx.Portability.ObjectModel;
+using Microsoft.WindowsAzure.Storage;
+using System;
+
+namespace Functions
+{
+    public class InjectConfiguration : IExtensionConfigProvider
+    {
+        public virtual void Initialize(ExtensionConfigContext context)
+        {
+            var services = new ServiceCollection();
+            RegisterServices(services);
+            var serviceProvider = services.BuildServiceProvider(true);
+
+            context
+                .AddBindingRule<InjectAttribute>()
+                .Bind(new InjectBindingProvider(serviceProvider));
+
+            var registry = context.Config.GetService<IExtensionRegistry>();
+            var filter = new ScopeCleanupFilter();
+            registry.RegisterExtension(typeof(IFunctionInvocationFilter), filter);
```

Comment:
nit: grouping variable declarations/object creation together and an empty new line after before function calls.

I think it helps with understanding where all the things are created and when they start getting used.

Reply:
Fixed

---
In reply to: [178090368](https://github.com/Microsoft/dotnet-apiport/pull/598#discussion_r178090368) [](ancestors = 178090368)